.github/CHANGELOG.md

```diff
@@ -14,6 +19,8 @@
 
 This release contains contributions from (in alphabetical order):
 
+Nathan Killoran
```

Comment:
lol

Reply:
Been busy today. Lot's of PRs to update!